packages/react-widgets/src/Calendar.tsx

```diff
@@ -444,8 +441,8 @@ function Calendar({
         e.preventDefault()
         navigate(direction)
       } else {
-        if (isRtl && OPPOSITE_DIRECTION[direction])
-          direction = OPPOSITE_DIRECTION[direction]
+        if (isRtl && (OPPOSITE_DIRECTION as any)[direction])
+          direction = (OPPOSITE_DIRECTION as any)[direction]
```

Comment:
shouldn't the types just work here if its typed as a Record?

Reply:
Fixed